controllers/storagecluster/cephcluster.go

```diff
@@ -445,6 +445,7 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string, kmsConfigMap *co
 				ManagePodBudgets:                 true,
 				ManageMachineDisruptionBudgets:   false,
 				MachineDisruptionBudgetNamespace: "openshift-machine-api",
+				OSDMaintenanceTimeout:            getOsdMaintenanceTimeout(sc),
```

Comment:
I was thinking this would allow us to set a different default under ocs-operator itself, if needed and we won't have to change the rook default.
But, if you suggest we let rook only take care of the defaults then I will remove the default setting from here.

Reply:
Updated the code to remove the default setting from ocs-operator.

Reply:
If there is a need for a separate default, that would be fine to declare it here, it just didn't seem necessary. At the same time, I don't expect the Rook default to change either.